### PR TITLE
add hycov plugin

### PR DESCRIPTION
**Pull request by DreamMaoMao**
Can I add hycov plugin into awesome-hyprland?
I want to share it with more people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Awesome list for Hyprland, that includes useful tools and libraries that either
 - [hyprland-virtual-desktops](https://github.com/levnikmyskin/hyprland-virtual-desktops) ![c++][cpp] (manages multiple screen's workspaces as if they were a single virtual desktop)
 - [Hypr-DarkWindow](https://github.com/micha4w/Hypr-DarkWindow) ![c++][cpp] (Hyprland plugin that adds possibility to invert the color of specific windows)
 - [grab-workspace](https://github.com/CMurtagh-LGTM/grab-workspace) ![c++][cpp] (plugin that moves a workspace to the current monitor and shows it)
+- [hycov](https://github.com/DreamMaoMao/hycov) ![c++][cpp] (plugin that overview all clients in grid layout)
```
Comment (yavko) [inline README.md:36]: The grammar is kinda off, "Hyprland plugin that displays all open clients in a grid view" sounds better imo

**Review by yavko — APPROVED** [review]

**Comment by yavko** [comment]
I meant to request changed oopsie

**Comment by DreamMaoMao** [comment]
done

**Comment by DreamMaoMao** [comment]
Is there anything else that needs fixing?

**Review by yavko — APPROVED** [review]

**Pull request merged**